.github/workflows/ci.yml

```diff
@@ -20,7 +20,21 @@ jobs:
         php-version: ${{ matrix.php }}
         coverage: xdebug
         extensions: ${{ matrix.extension }}
-    - run: composer install --prefer-dist --no-interaction
+        tools: composer:v2
+    - name: Install PHP 8 Dependencies
+      uses: nick-invision/retry@v1
+      with:
+        timeout_minutes: 5
+        max_attempts: 5
+        command: composer update --no-interaction --no-progress --ignore-platform-req=php
```

Comment:
Why do we ignore the platform? Is is because it fails on PHP 8?

Reply:
Yes because some dependencies doesn't allow php 8 yet